tests/test_tensor_flow_block.py

```diff
@@ -16,13 +16,13 @@ class TestTensorFlowBlock(NIOBlockTestCase, tf.test.TestCase):
 
     def output_dict(self, input_id):
         sig = {'loss': ANY if input_id != 'predict' else None,
-               'accuracy': ANY if input_id != 'predict' else None,
-               'prediction': ANY if input_id == 'predict' else None,
+               'prediction': ANY,
                'input_id': input_id}
         return sig
 
     @patch('tensorflow.Session')
-    def test_process_train_signals(self, mock_sess):
+    @patch('tensorflow.summary')
```

Comment:
Not sure why this was added, not being used in this test.

Reply:
I think these were left over from a point in time where tensorboard was always on, so it was necessary to patch out `summary` to avoid writing files during testing. Looks like it can be removed from all tests where `tensorboard_int` config is 0. good catch.